components/eamxx/src/dynamics/homme/eamxx_homme_iop.cpp

```diff
@@ -333,67 +338,60 @@ apply_iop_forcing(const Real dt)
   // TODO: Create a memory buffer for this class
   //       and add the below WSM and views
   WorkspaceMgr eamxx_wsm(NLEVI, 7+qsize, policy_eamxx);
-  WorkspaceMgr homme_wsm(NLEV, 16 + (theta_hydrostatic_mode ? 16 : 0), policy_homme);
   view_Nd<Pack, 4>
-    rstar      ("rstar",       nelem, NGP, NGP, NLEV),
-    exner      ("exner",       nelem, NGP, NGP, NLEV),
     temperature("temperature", nelem, NGP, NGP, NLEV);
 
-  // Lambda for computing rstar, exner, and temperature from Hommexx
+  // Lambda for computing temperature from Hommexx
   auto compute_homme_states = [&] () {
```

Comment:
We're no longer computing multiple homme states, I would change the name here to something like `compute_temperature`. I would also remove "from Hommexx" in the comment, since we are using physics functions.